app/src/main/java/com/duckduckgo/app/privacy/ui/TrackerNetworksViewModel.kt

```diff
@@ -16,18 +16,23 @@
 
 package com.duckduckgo.app.privacy.ui
 
+import android.net.Uri
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
+import com.duckduckgo.app.global.baseHost
 import com.duckduckgo.app.global.model.Site
+import com.duckduckgo.app.trackerdetection.model.Entity
+import com.duckduckgo.app.trackerdetection.model.TdsEntity
 import com.duckduckgo.app.trackerdetection.model.TrackingEvent
+import java.util.*
 
 class TrackerNetworksViewModel : ViewModel() {
 
     data class ViewState(
         val domain: String,
         val allTrackersBlocked: Boolean,
         val trackerCount: Int,
-        val trackingEventsByNetwork: Map<String, List<TrackingEvent>>
+        val trackingEventsByNetwork: SortedMap<Entity, List<TrackingEvent>>
```

Comment:
This data structure only exists for display purposes and since we display these as a **sorted** list of entities (with major trackers coming first) then it felt like a better fit for storage. We used to keep a separate list of sorted keys to allow us to pull items out of the map in the right order which made the logic harder to follow.